### PR TITLE
Participation moods pie widget documentation

### PR DESCRIPTION
**Pull request by th1988**
Hey @SweeD,

please review.

https://app.asana.com/0/906408746385033/906485324079586/f
https://app.asana.com/0/906408746385033/906485324079612/f

### CODE REVIEW

File: source/includes/_widgets.md [hunk 1]
```diff
@@ -204,6 +204,94 @@ to                  | End date of the tracking period
...
+{
+  "data": {
+    "id": "company-participation",
+    "type": "widgets",
+    "attributes": {
+      "moods_count": 2,
+      "employee_amount": 10,
+      "participation_in_percent": 12,
+      "moods_in_percent": [
+        { "feeling": "sad", "percent": 50 },
```
Comment (SweeD) [inline source/includes/_widgets.md]: I would do it a bit different. 

For example:
```
{
  moods_details: [
    { "feeling": "happy", "percent": 50, "count": 5 }, //...
  ]
}
```

This way, we can add details and also count, etc. 
What do you think?

File: source/includes/_widgets.md [hunk 2]
```diff
@@ -204,6 +204,94 @@ to                  | End date of the tracking period
...
+
+{
+  "data": {
+    "id": "department-participation",
+    "type": "widgets",
+    "attributes": {
+      "moods_count": 2,
+      "employee_amount": 10,
+      "participation_in_percent": 12,
+      "moods_in_percent": [
```
Comment (SweeD) [inline source/includes/_widgets.md]: Same like comment before.

File: source/includes/_widgets.md (hunk 1 above)
Comment (th1988) [reply source/includes/_widgets.md]: Good idea. I'll change it asap 👍

**Comment by th1988** [comment]
Changes done @SweeD

**Pull request merged**